src/components/atoms/videoTitle/VideoTitle.tsx

```diff
@@ -7,7 +7,7 @@ type VideoTitleProps = {
 
 export function VideoTitle({ title, sx }: VideoTitleProps) {
   return (
-    <Typography variant="h3" sx={{ ...sx, textTransform: 'uppercase' }}>
+    <Typography variant="h3" sx={{ ...sx }}>
```

Comment:
Is the text transform gone because of the font? It looks like video titles are supposed to be all caps.